travelpal/assets/js/components/BookedCard.jsx

```diff
@@ -1,39 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Card, CardBody, Button, Row, Col } from 'reactstrap';
+import { FormGroup, Input, Label } from 'reactstrap';
 
 import api from '../api';
 import TripCardHeader from './TripCardHeader';
+import BookedForm from './BookedForm';
 
 // Renders the details of an individual booked trip
-export default function BookedCard({destination, startDate, endDate, price,
-  id}) {
+export default function BookedCard({form, destination, startDate, endDate,
```

Comment:
Yeah... might change it back to props

Reply:
Wait, I don't think you were implying to change it back to props. How would you reorganize this?

Also as a side note: I'm still more focused on functionality than code cleanup atm.

Reply:
that would be more work to change it back to props. I mean reorganize it in such away where you can just pass in {...bookedCardProps}

Reply:
@kimberlypn  I'm not too sure what the exact clean up would since I just finished AI and i'll be working on this tmr. I'll let you know when I come up with the prop types fix.

Reply:
Hm, I'll just pass the entire object (the {...tt} like you said in a later comment).

Reply:
@Guo-William Okay, I think I was able to clean it up! You can take a look.